### ISSUE TITLE
make android.debug fails to build

### ISSUE THREAD

**Issue by flavienbwk** [berty/berty#3182]
Hi,

I am trying to build berty's app but am encoutering this error : 

```bash
~/berty-local/berty/js$ make android.debug

cd .. && go  mod download
cd .. && go  run golang.org/x/mobile/cmd/gomobile init
mkdir -p android/libs
cd .. && GO111MODULE=on go  run golang.org/x/mobile/cmd/gomobile bind \
        -o js/android/libs/gobridge.aar \
        -v -ldflags="-X berty.tech/berty/v2/go/pkg/bertyversion.VcsRef=`git rev-parse --short HEAD` -X berty.tech/berty/v2/go/pkg/bertyversion.Version=`go  run -mod=readonly -modfile=/home/myuser/berty-local/berty/js/../go.mod github.com/mdomke/git-semver/v5`" \
        -cache "/home/myuser/berty-local/berty/js/android/.gomobile-cache" \
        -target android \
        -androidapi 21 \
        ./go/framework/bertybridge
write /home/myuser/berty-local/berty/js/android/.gomobile-cache/src/go.mod
go: berty.tech/go-ipfs-log@v1.2.6: missing go.sum entry; to add it:
        go mod download berty.tech/go-ipfs-log
/tmp/go-build139665614/b001/exe/gomobile: go build -v -ldflags -X berty.tech/berty/v2/go/pkg/bertyversion.VcsRef=96b1106d -X berty.tech/berty/v2/go/pkg/bertyversion.Version=v2.263.1-dev.2+g96b1106d -buildmode=c-shared -o=/home/myuser/berty-local/berty/js/android/.gomobile-cache/android/src/main/jniLibs/armeabi-v7a/libgojni.so ./gobind failed: exit status 1

exit status 1
Makefile:413: recipe for target 'android/libs/gobridge.aar' failed
make: *** [android/libs/gobridge.aar] Error 1
``` 

Could you tell me what to do ?

Thanks,

**Comment by aeddi** [comment]
Hi,

You are probably using go in version 1.16, which was incompatible until now.
Try building again after pulling master, we just merged a fix: #3106 ;)

**Comment by flavienbwk** [comment]
Problem solved, thanks !
